testing/kfctl/kf_is_ready_test.py

```diff
@@ -60,12 +63,12 @@ def test_kf_is_ready(namespace, use_basic_auth, use_istio):
   # TODO(jlewi): Might want to parallelize this.
   for deployment_name in deployment_names:
     logging.info("Verifying that deployment %s started...", deployment_name)
-    util.wait_for_deployment(api_client, namespace, deployment_name)
-  
+    util.wait_for_deployment(api_client, namespace, deployment_name, 10)
```

Comment:
what does 10 mean here?

Reply:
It means timeout in 10mins if the deployment hasn't started yet.